### PR TITLE
Update bangalore office new address

### PR DESCRIPTION
**Pull request by priyanka1110**
## Description
This PR fixes the Issue - [Bangalore address is not updated](https://github.com/Infratab/Website-IN/issues/25)
Note: This issue is already merged for this PR #27. I am referring to closed issues As the release didn't happen in Jan and again office address got updated.

# Implementation
***Contact info in the contact page***

Updated the Bangalore office new address and address on the map tooltip with the location co-ordinates. 

From 
```
 Infratab Bangalore Pvt Ltd
 Cowrks, Building 6A,
 RMZ Eco World, Outer Ring Rd,
 Devarabisanahalli, Bellandur, 
 Bengaluru, Karnataka 560103
```
To: 
```
Infratab Bangalore Pvt Ltd
91 springboard, 2nd Floor,
Padmavathi Complex, 80 Feet Rd,
Koramangala 8th Block,
Bengaluru-560095, Karnataka
```

## Screenshots
*Address*
Contact

*Address in tooltip*
Contact-map

## Related PRs/Issues
List related PRs against other branches:

Issue Title | PR URL
-------|-----
Bangalore address is not updated | #25 

## Impacted Areas
List related PRs from other repositories
- No impact on other repositories

List expected works in other repositories
- No work in other repositories

### Checklist
- [x] Set proper milestone and labels
- [x] Code compiles correctly
- [x] Lint pass locally with my changes
- [x] Extended the README/documentation as necessary
- [x] Test Summary - https://infratab.atlassian.net/wiki/spaces/TC/pages/374407418/31+Update+Bangalore+office+new+address

### CODE REVIEW

**Comment by jovials146** [comment]
@priyanka1110 Did you updated map coordinates?

**Comment by priyanka1110** [comment]
@jovials146 I have updated the coordinates also.

**Review by jovials146 — APPROVED** [review]
LGTM

**Review by rahulkarkera — APPROVED** [review]
LGTM

File: contact.html [hunk 1]
```diff
@@ -54,10 +54,9 @@ <h2 class="location-header">India Development Center</h2>
           <div class="address">
             <i class="icon-location icon"></i>
             <p>
-               <strong> Infratab Bangalore Pvt Ltd </strong><br>
-               	Cowrks, Building 6A,<br> RMZ Eco World,
-                Outer Ring Rd,<br> Devarabisanahalli,
-                Bellandur, <br> Bengaluru, Karnataka 560103<br>
+              <strong> Infratab Bangalore Pvt Ltd </strong><br>
+               91 springboard, 3nd Floor,<br> Padmavathi Complex, 80 Feet Rd,
```
Comment (niren777) [inline contact.html]: @priyanka0901 should we use 2nd floor here?

Comment (priyanka1110) [reply contact.html]: @niren777 Because office address is registered with 2nd floor.

**Review by rahulkarkera — APPROVED** [review]
LGTM

**Review by niren777 — APPROVED** [review]

**Pull request merged**